[2025-02-02T22:00:41.417+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-02T22:00:41.450+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: flights_pipeline.load_data_into_duckdb manual__2025-02-02T22:00:24.944684+00:00 [queued]>
[2025-02-02T22:00:41.483+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: flights_pipeline.load_data_into_duckdb manual__2025-02-02T22:00:24.944684+00:00 [queued]>
[2025-02-02T22:00:41.487+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 2
[2025-02-02T22:00:41.520+0000] {taskinstance.py:2879} INFO - Executing <Task(_PythonDecoratedOperator): load_data_into_duckdb> on 2025-02-02 22:00:24.944684+00:00
[2025-02-02T22:00:41.535+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=7793) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-02T22:00:41.539+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'flights_pipeline', 'load_data_into_duckdb', 'manual__2025-02-02T22:00:24.944684+00:00', '--job-id', '128', '--raw', '--subdir', 'DAGS_FOLDER/zdag.py', '--cfg-path', '/tmp/tmppt9ad73g']
[2025-02-02T22:00:41.541+0000] {standard_task_runner.py:72} INFO - Started process 7825 to run task
[2025-02-02T22:00:41.543+0000] {standard_task_runner.py:105} INFO - Job 128: Subtask load_data_into_duckdb
[2025-02-02T22:00:41.618+0000] {task_command.py:467} INFO - Running <TaskInstance: flights_pipeline.load_data_into_duckdb manual__2025-02-02T22:00:24.944684+00:00 [running]> on host d628bd12729b
[2025-02-02T22:00:41.769+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='flights_pipeline' AIRFLOW_CTX_TASK_ID='load_data_into_duckdb' AIRFLOW_CTX_EXECUTION_DATE='2025-02-02T22:00:24.944684+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-02T22:00:24.944684+00:00'
[2025-02-02T22:00:41.771+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-02T22:00:41.847+0000] {logging_mixin.py:190} INFO - Données chargées dans DuckDB avec succès.
[2025-02-02T22:00:41.854+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-02T22:00:41.872+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-02T22:00:41.873+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=flights_pipeline, task_id=load_data_into_duckdb, run_id=manual__2025-02-02T22:00:24.944684+00:00, execution_date=20250202T220024, start_date=20250202T220041, end_date=20250202T220041
[2025-02-02T22:00:41.924+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2025-02-02T22:00:41.959+0000] {taskinstance.py:3891} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2025-02-02T22:00:41.964+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
